tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "exclude": ["example"],
```

Comment:
Why did you add this here? Now the `example/` doesn't do any typechecking because it doesn't have it's own tsconfig, no?

Reply:
I added a `example/tsconfig.json` file; `yarn tsc` at the root level throws type errors when it type-checks the `example/**/*.test.ts` files because the Jest types are only defined in `example/node_modules`, so figured it's neater to have separate tsconfig files.

Reply:
Just realised that the "Run TypeScript in example/" job wasn't running because `typescript` script wasn't defined in `example/package.json` - updated that